src/yadmark/benchmark/external/apfel_utils.py

```diff
@@ -31,6 +31,9 @@ def load_apfel(theory, observables, pdf="ToyLH", use_external_grid=False):
     apfel.SetPolarizationDIS(observables.get("PolarizationDIS", 0))
     apfel.SetProjectileDIS(observables.get("ProjectileDIS", "electron"))
     apfel.SetTargetDIS(observables.get("TargetDIS", "proton"))
+    charge = observables.get("NCPositivityCharge")
+    if charge is not None:
+        apfel.SelectCharge(charge)
```

Comment:
well, I wanted to exactly improve over the APFEL name because "Charge" is no option - so I went for "PositivityCharge" to make the usage explicit and then realized that it only acts in NC and added that as further explicit bit ... if you have a better name please ;-)